### DISCUSSION TITLE
Introducing Myself

### DISCUSSION THREAD

**Discussion by dnabanita7** [OpenMined/courses#138] [Introduce Yourself]
Hello Everyone :wave: This is Nabanita Dash, a fellow privacy enthusiast. I am mostly interested in differential privacy and want to contribute in pyDP. I am taking this course to help me in that regard. I am a 4th year undergraduate at IIIT, BBSR, India majoring in Comp Sci. I hope to learn a lot and have fun along the way.

**Comment by yemikifouly** [comment]
Hello Nabanita! 👋  Welcome to OpenMined and the discussion board! Have you started the course yet? If yes, I hope you are enjoying it very much. I love that you have a clear goal and wish to contribute to PyDP. 🙂 Have you chatted with @Kritikalcoder, the Differential Privacy Team Lead yet? You can also find here in Slack with the username @Kritika.
